examples/svelte-apollo/src/Articles.svelte

```diff
@@ -0,0 +1,44 @@
+<script context="module">
+  import { gql } from '@apollo/client'
+  import { client } from './apollo'
+
+  export const ARTICLES = gql`
```

Comment:
Sometimes we're using `ARTICLE_LIST` and sometimes `ARTICLES`. I think we should use one or the other more consistently. I would suggest using the plural form for an entity.

So I would rename `ARTICLE_LIST` to `ARTICLES` and `AUTHROR_LIST` to `AUTHORS` everywhere.

Reply:
Done renaming to plural forms for the entities.